ecosystem/windows_support.md

```diff
@@ -1,3 +1,27 @@
 # Windows Support
 
-TODO
+## Installation
+
+Various projects exist for installing OCaml under windows. We recommend sticking to 
+[OCaml for Windows][ocaml-win] as it is the most recent distribution.
+
+* [OCaml for Windows][ocaml-win] - The most maintained version, which installs the Cygwin toolchain 
+and OCaml on top of that. The OCaml versions lag slightly behind, but not as much as the other projects.
+* [OCPWin][ocaml-ocpwin] - A self-contained binary distribution for Windows, supports MSVC. This distribution 
+lags behind on OCaml versions 
+* [Installing from source][ocaml-from-source] - Its possible to also install OCaml manually using the 
+Cygwin toolcheck by following this README, published by the OCaml project.
+
+
+## Limitations
+
+The versions supported by the windows distributions seem to lag behind the versions 
+supported on Unix. Many tools (`ocamlbuild`, `ocamlfind`, etc.) can also not be used 
```

Comment:
`can't be used` is probably better English.

Reply:
Fixed, thanks